src/libraries/Microsoft.PowerFx.Connectors/ConnectorFunction.cs

```diff
@@ -292,15 +294,23 @@ public ConnectorType ReturnParameterType
         // Those properties are only used by HttpFunctionInvoker
         internal ConnectorParameterInternals _internals = null;
 
-        private readonly ConnectorLogger _configurationLogger = null;
+        private readonly ConnectorLogger _configurationLogger = null;
+
+        internal readonly OptionSetList OptionSets;
+
+        // CDP only
+        internal readonly string TableName;
```

Comment:
This seems strange - 
Why would CDP use ConnectorFunction at all? CDP can directly call the endpoint.

Reply:
This is used in GetCdpTableType even though we don't use the connection function
It is there as we share existing code (used for dynamic intellisense)
As explained during our call, the CDP schema is a swagger that defines types (tables) and no API and we share the same code/extensions

Reply:
Can be removed